ui/src/components/admin/navigation/Sidebar.js

```diff
@@ -0,0 +1,270 @@
+import React from "react";
+import { useHistory } from "react-router";
+import {
+  IconButton,
+  Avatar,
+  Box,
+  CloseButton,
+  Flex,
+  HStack,
+  VStack,
+  Icon,
+  useColorModeValue,
+  Link,
+  Drawer,
+  DrawerContent,
+  Text,
+  useDisclosure,
+  BoxProps,
+  FlexProps,
+  Menu,
+  MenuButton,
+  MenuDivider,
+  MenuItem,
+  MenuList,
+  Button,
+  useColorMode,
+} from "@chakra-ui/react";
+import {
+  FiHome,
+  FiTrendingUp,
+  FiCompass,
+  FiStar,
+  FiSettings,
+  FiMenu,
+  FiBell,
+  FiChevronDown,
+  FiAperture,
+} from "react-icons/fi";
+import AdminUser from "../users/AdminUser";
+
+//import { Switch, Route, Redirect } from "react-router-dom";
+//import { IconType } from "react-icons";
+//import { ReactText } from "react";
+
+// interface LinkItemProps {
+//   name: string;
+//   icon: IconType;
+// }
+const LinkItems = [
+  { name: "Home", icon: FiHome, path: "/admin/home" },
+  {
+    name: "Users",
+    icon: FiTrendingUp,
+    path: "/admin/users",
+    component: "AdminUser",
+  },
+  { name: "Forms", icon: FiCompass, path: "/admin/forms" },
+  { name: "Giving", icon: FiStar, path: "/admin/giving" },
+  { name: "Settings", icon: FiSettings, path: "/admin/settings" },
+];
+
+const SidebarWithHeader = ({ children, ...rest }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    // <Box>
+    <>
+      <SidebarContent
```

Comment:
i dont get it. why is there a `SidebarContent` component here, and then another one inside `DrawerContent`?